### PR TITLE
(maint) Updated Octokit NuGet package

### PR DESCRIPTION
**Pull request by gep13**
- Let Visual Studio do it's thing with a couple project files

NOTE: First attempt at correcting problem with Package Verifier currently executing.

### CODE REVIEW

File: src/chocolatey.package.verifier/app.config [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Reactive.Linq" publicKeyToken="94bc3704cddfc263" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-3.0.1000.0" newVersion="3.0.1000.0" />
+      </dependentAssembly>
+    </assemblyBinding>
+  </runtime>
+</configuration>
```
Comment (ferventcoder) [inline src/chocolatey.package.verifier/app.config]: This one is probably unnecessary.

Comment (ferventcoder) [reply src/chocolatey.package.verifier/app.config]: That's okay though, I'm not worried about it at this point.

**Comment by ferventcoder** [comment]
🤞 this fixes the issue.

**Comment by gep13** [comment]
SWEET! 😄

File: src/chocolatey.package.verifier/app.config (hunk 1 above)
Comment (gep13) [reply src/chocolatey.package.verifier/app.config]: @ferventcoder yeah, I debated about it to be honest.  Part of me was like "I am sure that Visual Studio and NuGet knows what they are doing", and then I was also like "on the other hand..." 😄

**Pull request merged**